### PR TITLE
Update unit tests

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

 none

### Dependencies

 none

### Description

Stop using deprecated methods in our unit tests.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204527258277760

### CODE REVIEW

File: test/core/currency/wallet/currency-wallet.test.ts [hunk 1]
```diff
@@ -210,28 +211,25 @@ describe('currency wallets', function () {
...
-    log.assert('customTokens changed')
+      displayName: 'Custom Token',
+      denominations: [{ multiplier: '1000', name: 'CUSTOM' }],
+      networkLocation: {
+        contractAddress:
+          '0X7CD5885327FD60E825D67D32F9D22B018227A208AA3C4819DA15B36B5D5869D3'
+      }
+    }
 
+    await config.addCustomToken(customToken)
```
Comment (samholmes) [inline test/core/currency/wallet/currency-wallet.test.ts:229]: The customToken parameter here looks different from before. How is the networkLocation able to change without implementation change; are both network locations values cleaned to the same object in the assertion?

File: src/core/fake/fake-world.ts [hunk 2]
```diff
@@ -144,11 +144,14 @@ export function makeFakeWorld(
...
 
       // Figure out which repos to use:
-      const syncKeys = account.allKeys
-        .filter(info => info.keys?.syncKey != null)
-        .map(info =>
-          base16.stringify(base64.parse(info.keys.syncKey)).toLowerCase()
+      const syncKeys: string[] = []
+      for (const info of account.allKeys) {
+        const keys = await account.getRawPrivateKey(info.id)
+        if (keys?.syncKey == null) continue
```
Comment (samholmes) [inline src/core/fake/fake-world.ts:150]: No need for `?` now, I assume.

File: test/core/currency/wallet/currency-wallet.test.ts (hunk 1 above)
Comment (swansontec) [reply test/core/currency/wallet/currency-wallet.test.ts:229]: No, these are different functions. The `EdgeCurrencyConfig` version takes an `EdgeToken`, but the `EdgeCurrencyWallet` one takes an `EdgeMetaToken`, so the format has to change.

File: src/core/fake/fake-world.ts (hunk 2 above)
Comment (swansontec) [reply src/core/fake/fake-world.ts:150]: Good catch.

**Pull request merged**